src/api/SnykApiClient.ts

```diff
@@ -235,7 +235,7 @@ export class SnykApiClient implements SnykApi {
     async listAllAggregatedIssues(orgId: string, projectId: string): Promise<UnifiedIssues> {
         if (this.isMocked()) {
             await new Promise((resolve) => setTimeout(resolve, 500));
-            return mockedIssues[projectId];
+            return mockedIssues[projectId].data;
```

Comment:
Fix for:

<img width="1042" height="39" alt="Image" src="https://github.com/user-attachments/assets/20b80058-0cc7-4a55-81fe-5279d3250a8a" />